monosi/anomalies/__init__.py

```diff
@@ -30,8 +30,11 @@ def _std_dev(cls, values: List[Any]):
     def _z_scores(cls, stats: List[Any]):
         values = [stat.value for stat in stats]
 
-        mean = cls._mean(values)
-        std_dev = cls._std_dev(values)
+        try:
+            mean = cls._mean(values)
+            std_dev = cls._std_dev(values)
+        except:
```

Comment:
Should only catch divide by zero exception

Reply:
added `except ZeroDivisionError:`